bin/pos-cli-modules-install.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+
+const program = require('commander'),
+  degit = require('degit');
+
+const logger = require('../lib/logger'),
+  report = require('../lib/logger/report'),
+  dir = require('../lib/directories');
+
+function cloneModule(moduleName) {
+  const moduleRepo = `Platform-OS/pos-module-${moduleName}`;
```

Comment:
@hosszukalman I'm not sure we want this prefix to be hardcoded :/

Reply:
If I remember correctly we already agreed on that prefix. We can change it later, but now we don't have a separate organization or something like that to collect our modules.